src/styles/nav.css

```diff
@@ -0,0 +1,119 @@
+/* ---- base ---- */
```

Comment:
when I rename it .scss it all breaks, whether I rename the file in import or not